public/sw_global_variables.js

```diff
@@ -8,13 +8,18 @@
 const cacheVersion = 'cache v1.2';
 
 // API Source
-const apiSource = 'https://360apitrain.gordon.edu/api';
+// @PROD
+// const apiSource = 'https://360Api.gordon.edu/api';
```

Comment:
Just checking - is the source supposed to be Train?

Reply:
Yes, the API is currently Train. The @TRAIN comment is right below lol. I have the Production source commented out so it's easier to change the source between Production and Development.